driver/base/response.go

```diff
@@ -17,7 +17,35 @@ var ErrFailedOpeningTemplate = errors.New("failed opening provided path to textf
 // ErrFailedParsingTemplate error for failure of parsing a textfsm template.
 var ErrFailedParsingTemplate = errors.New("failed parsing textfsm template")
 
-// Response response object that gets returned from scrapli send operations.
+func stringSliceContains(s string, l []string) (bool, string) { //nolint:gocritic
```

Comment:
Worth moving to utils?

Reply:
totally.... when I was looking at this yesterday I think I realized there are a handful of little things like this that are maybe even duplicated... will take a peak!